common/workflows/src/index.ts

```diff
@@ -0,0 +1,251 @@
+///////////////////////////////////////////////////////////////////////////////
+// Workflow
+///////////////////////////////////////////////////////////////////////////////
+export interface Workflow {
+    id: string
+    name: string
+    description: string
+    json: {
```

Comment:
`json` fields weird. It should be "workflow" or "config" or something indicating what its for